extensions/ql-vscode/package.json

```diff
@@ -461,13 +461,17 @@
         "command": "codeQLQueryHistory.showQueryText",
         "title": "Show Query Text"
       },
+      {
+        "command": "codeQLQueryHistory.exportCsvResults",
+        "title": "Export Results (CSV)"
```

Comment:
From a user's perspective (especially a beginner user), it might be confusing that for alert data, you "view" the results, but for non-alert data, you "export" the results. 

Here is one suggestion. Rename this command (or "View Alerts (CSV)") so that they are the same. And for this command instead of asking the user to choose a file name, auto-generate a file name and place it next to the BQRS file. This way, it feels the same to the user (even though under the hood, it is a different action). 

This is just a suggestion and I'm fine with it going in as it is.

Reply:
Thanks, that does seem more intuitive. I have changed it to do that!